src/h5web/toolbar/controls/DomainSlider.tsx

```diff
@@ -23,6 +24,8 @@ function DomainSlider(props: Props): ReactElement {
   const [extendedMin, extendedMax] = extendDomain(dataDomain, EXTEND_FACTOR);
   const step = Math.max((extendedMax - extendedMin) / 100, 10 ** -NB_DECIMALS);
 
+  const isAutoScaling = value.some(isUndefined);
```

Comment:
You might want to use the `isDefined` already defined in `h5web/guards`

Reply:
Or:
```js
const isAutoMin = value[0] === undefined && value[1] !== undefined
const isAutoMax = value[0] !== undefined && value[1] === undefined
const isAutoScaling = isAutoMin || isAutoMax
```

and then, you can use `isAutoMin` and `isAutoMax` to make the nested ternary below more readable

Reply:
Good idea, though in your suggestion, `isAutoScaling` doesn't end up with the same value. This works great, however:

```ts
const isAutoMin = value[0] === undefined;
const isAutoMax = value[1] === undefined;
const isAutoScaling = isAutoMin || isAutoMax;
```